src/App.js

```diff
@@ -1,24 +1,45 @@
-import logo from './logo.svg';
 import './App.css';
+import './fonts/Hind-Regular.ttf'
+import './fonts/Montserrat-VariableFont_wght.ttf'
+import Header from './components/Header'; 
+import Form from './components/Form';
+import Gallery from './components/Gallery'
+import Footer from './components/Footer';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
 
 function App() {
+
+  const [recipes, setRecipes] = useState([]);
+  const [inputValue, setInputValue] = useState(' ');
+
+  useEffect(() => {
+    axios({
+      baseURL: 'https://api.spoonacular.com/recipes/complexSearch',
+      params: {
+        apiKey: 'cf6c7cb9ef804ec5a5ee4edd6f19b1ee',
+        query: inputValue,
+        number: 20,
+        addRecipeInformation: true
+      }
+    }).then((apiData) => {
+      setRecipes(apiData.data.results)      
+    })
+
+  }, [inputValue]);
+
+  const selectInputValue = (event, chosenInput) => {
+        event.preventDefault();
+        setInputValue(chosenInput);
+    }
```

Comment:
Good, simple input function! The indentation on line 34 is a bit inconsistent though (4 spaces vs. 2)